src/server/VRxControl.py

```diff
@@ -18,25 +18,31 @@ def __init__(self, RHData, Events, RACE, nodes, Language, **_kwargs):
         self.nodes = nodes
         self.Language = Language
 
-        self.enabled = True
+        self.enabled = None
         self.controllers = {} # collection of VRxControllers
 
         self.Events.trigger('VRxC_Initialize', {
             'registerFn': self.registerController
             })
 
-        self.Events.on(Evt.STARTUP, 'VRx', self.doStartup)
-        self.Events.on(Evt.HEAT_SET, 'VRx', self.doHeatSet)
-        self.Events.on(Evt.RACE_STAGE, 'VRx', self.doRaceStage, {}, 75)
-        self.Events.on(Evt.RACE_START, 'VRx', self.doRaceStart, {}, 75)
-        self.Events.on(Evt.RACE_FINISH, 'VRx', self.doRaceFinish)
-        self.Events.on(Evt.RACE_STOP, 'VRx', self.doRaceStop)
-        self.Events.on(Evt.RACE_LAP_RECORDED, 'VRx', self.doRaceLapRecorded, {}, 200, True)
-        self.Events.on(Evt.LAPS_CLEAR, 'VRx', self.doLapsClear)
-        self.Events.on(Evt.LAP_DELETE, 'VRx', self.doLapDelete)
-        self.Events.on(Evt.FREQUENCY_SET, 'VRx', self.doFrequencySet, {}, 200, True)
-        self.Events.on(Evt.MESSAGE_INTERRUPT, 'VRx', self.doSendPriorityMessage)
-        self.Events.on(Evt.OPTION_SET, 'VRx', self.doOptionSet)
+        if len(self.controllers):
```

Comment:
No need to check for len. Empty collections (dict or lists or tuples or sets) eval False.